src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -274,12 +274,14 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
             </p>;
         }
 
-        let e2eeSection;
+        let e2eeSection: JSX.Element;
         if (this.state.joinRule !== JoinRule.Public) {
-            let microcopy;
+            let microcopy: string;
             if (privateShouldBeEncrypted()) {
                 if (this.state.canChangeEncryption) {
-                    microcopy = _t("You can't disable this later. Bridges & most bots won't work yet.");
+                    microcopy = isVideoRoom
+                        ? _t("You can't disable this later. The room will be encrypted but the embedded call is not.")
```

Comment:
Maybe tweak the verb tenses ("will be" / "is not"). Also, if we're planning on rolling out encrypted Element Call eventually, do we want to add a "yet", or will that just be confusing?